README.md

```diff
@@ -104,6 +104,7 @@ dashboards as inspiration for your application.
 The exported metrics are:
 - `kafka_consumer_partition_delay`: number of records betweeen last record consumed successfully and the last record on kafka, by partition and topic.
 - `kafka_consumer_records_consumed_successfully`: number of records consumed successfully by this instance.
+- `kafka_consumer_endpoint_latency_histogram_seconds`: endpoint latency in seconds(insertion to elasticsearch).
```

Comment:
(Not too important): There's a space missing between the opening parenthesis and the word "seconds". This is not a function call.

Reply:
good catch, changing it